pages.ta/linux/tshark.md

```diff
@@ -0,0 +1,36 @@
+# tshark
+
+> பாக்கெட் பகுப்பாய்வு கருவி, வயர்ஷார்க்கின் CLI பதிப்பு.
+> மேலும் விவரத்திற்கு: <https://tshark.dev/>.
+
+- லோக்கல் ஹோஸ்டில் அனைத்தையும் கண்காணிக்கவும்:
+
+`tshark`
+
+- குறிப்பிட்ட பிடிப்பு வடிப்பானுடன் பொருந்தும் பாக்கெட்டுகளை மட்டும் பிடிக்கவும்:
+
+`tshark -f '{{udp port 53}}'`
+
+- குறிப்பிட்ட வெளியீட்டு வடிப்பானுடன் பொருந்தும் பாக்கெட்டுகளை மட்டும் காட்டு:
+
+`tshark -Y '{{http.request.method == "GET"}}'`
+
+- ஒரு குறிப்பிட்ட நெறிமுறையைப் பயன்படுத்தி TCP போர்ட்டை டிகோட் செய்யவும் (எ.கா. HTTP):
+
+`tshark -d tcp.port=={{8888}},{{http}}`
+
+- கைப்பற்றப்பட்ட வெளியீட்டின் வடிவமைப்பைக் குறிப்பிடவும்:
+
+`tshark -T {{json|text|ps|…}}`
+
+- வெளியீட்டிற்கு குறிப்பிட்ட புலங்களைத் தேர்ந்தெடுக்கவும்:
+
+`tshark -T {{fields|ek|json|pdml}} -e {{http.request.method}} -e {{ip.src}}`
+
+- கைப்பற்றப்பட்ட பாக்கெட்டை ஒரு கோப்பில் எழுதவும்:
+
+`tshark -w {{பாதை/டு/கோப்பு}}`
+
+- ஒரு கோப்பிலிருந்து பாக்கெட்டுகளை பகுப்பாய்வு செய்யுங்கள்:
+
+`tshark -r {{கோப்பு_பெயர்}}.pcap`
```

Comment:
```suggestion
`tshark -r {{பாதை/டு/கோப்பு.pcap}}`
```

Mandatory fix (error source: [original page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/tshark.md)): path placeholder is [broken](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).